lib/app/providers/user_provider.dart

```
@@ -0,0 +1,115 @@
+import 'package:flutter/cupertino.dart';
+import '../models/friend.model.dart';
+import '../models/user_model.dart';
+
+class UserProvider extends ChangeNotifier {
+  late List<User> users;
+  late Friend friend;
+  late List<User> friendsUsers, friendRequests, friendProposals;
+
+  Future<void> getUser() async {
+    // Simulate a network request
+    users = userTest;
+    notifyListeners();
+  }
+  Future<void> getFriendOfUser() async {
+    // Simulate a network request
+    friend = friendTest;
+    notifyListeners();
```

Mấy đứa dùng Android Studio không biết bật catch all exception à, log cũng đỏ lòm kìa